deploy/complete/helm-chart/provision/values.yaml

```diff
@@ -2,6 +2,12 @@ global:
   osb:
     # Set to true to create a global ATP instance
     atp: true
+    # Set to true to create a streaming service instance
+    oss: true
+    # Set to true to create a bucket on the object storage service instance
+    objectstorage: true
+    # Set object storage namespace, usualy the tenancy name or ocid.
+    objectstoragenamespace: ociateam
```

Comment:
shouldn't have our tenancy name here. In the comment, it states `ocid` can be used? Lastly, why not just nest this as:

```yaml
osb:
  objectstorage:
    namespace: mytenancy
```

Reply:
This was misplaced. Following the values-prod-reference.yaml on the mushop chart. For future changes, need to decide if will separate the creation of the bucket during the provision chart and the load of the images using the mushop chart or if will do all together.

Reply:
Sounds good. I am almost done with the OOS bucket auto-population, and the separation of the images (assets service) from `catalogue`.